lib/wallet/src/Cardano/Wallet/Write/Tx/Balance.hs

```diff
@@ -323,8 +323,7 @@ data UTxOAssumptions = forall k ktype. UTxOAssumptions
 -- either normal, post-Shelley credentials, or boostrap/byron credentials
 -- depending on the 'k' of the supplied 'TransactionLayer'.
 allKeyPaymentCredentials
-    :: forall k. TransactionLayer k 'CredFromKeyK SealedTx
-    -> UTxOAssumptions
```

Comment:
Can this be reverted? If you're calling `allKeyPaymentCredentials` then you should have `ktype ~ 'CredFromKeyK`. 

Secondly, I'd prefer to keep the existing formatting. While we do tend to write simple types on one line `foo :: Int -> Int`, I think it's in this case 1) a bit on the long side 2) breaks similarity to `allScriptPaymentCredentials`.

Reply:
I guess its a matter of personal taste (for me this formatting is confusing as it has type-level and value-level mixed in one line).

Reverted.

Reply:
Thanks.

> I guess its a matter of personal taste

While this is personal taste

>  I think it's in this case 1) a bit on the long side

this is not:

> 2) breaks similarity to allScriptPaymentCredentials.

— at least less directly so.